GestaoFinanceira/GestaoFinanceira/Controllers/EntryExpensesController.cs

```diff
@@ -35,11 +35,34 @@ public EntryExpenses Find(int entryId)
 
         public void Save(EntryExpenses entry)
         {
-            EntryExpenses ent = entry.Id != 0 ? Context.Expenses.First(c => c.Id == entry.Id) : null;
+            EntryExpenses ent = entry.Id != 0 ? Context.Expenses.First(e => e.Id == entry.Id) : null;

```

Comment:
Evite utilizar o `First`, pois, caso o elemento que você está buscando não se encontra na lista ele irá lançar uma exceção. Neste caso é preferível utilizar o `FirstOrDefault` que será retornando `null` quando não for encontrado o elemento.

Reply:
Bacana, Obrigado pelo apontamento!